ksql-common/src/main/java/io/confluent/ksql/util/KsqlStatementException.java

```diff
@@ -46,4 +46,9 @@ public String getRawMessage() {
   private static String buildMessage(final String message, final String sqlStatement) {
     return message + System.lineSeparator() + "Statement: " + sqlStatement;
   }
+
+  @Override
+  public String toString() {
+    return buildMessage(rawMessage, "<retracted>");
```

Comment:
I think you mean "redacted" rather than "retracted" everywhere, right?

Reply:
Redacting the statement text from KsqlStatementException looks like it defeats the purpose of having KsqlStatementException at all (rather than using the base KsqlException). I'm guessing this is just a temporary stop-gap for older branches, and that newer branches starting with 7.0.x will have different behavior. Is that true?

Reply:
Was following the same wording as our recent 0.27 release: https://github.com/confluentinc/ksql/commit/f60c93ed1cec86dede7a7beab13a04ff833138c6#diff-b36ea883a817613e501615df160333e11f9b7bf5bac2d0a8f9b3a9d235e48003R52

You are right, I want to just stop-gap for older branches up to 6.x and then have a cleaner solution for master / 7.x

Reply:
haha. I was also confused by `redacted` and `retracted`. I thought I saw `redacted` in @mjsax 's PR but I checked again, it's `retracted`...

Reply:
I hate to be pedantic about English but the word really should be "redacted" and not "retracted" -- "retracted" simply doesn't mean what we're using it to mean here. Can we update here, and also on later branches (whenever we get there)? We don't have to bother with fixing 0.27 since that's already out the door, but it'd be good to fix this going forward.